server/src/resolvers/query.js

```diff
@@ -1,10 +1,174 @@
+        maxParticipants: 1000,
+        tags: [],
+        participants: [],
+      },
+      {
+        id: 2,
+        host: 0,
+        creationTime: '',
+        lastUpdatedTime: '',
+        schedule: [],
+        title: 'What is meaning of life?',
+        description: 'Sometimes we thinking about huge size of the universe. ' +
+            'Compare to what humanity accomplished so far,' +
+            ' the universe seems wast of space... isn\' it?',
+        price: 999999.99,
+        maxParticipants: 1000,
+        tags: [],
+        participants: [],
+      },
+      {
+        id: 3,
+        host: 0,
+        creationTime: '',
+        lastUpdatedTime: '',
+        schedule: [],
+        title: 'If AI gets self awareness...',
+        description: 'That means bad thing to human? YES.' +
+            ' But in a perspective of evolution? NO.',
+        price: 999999.99,
+        maxParticipants: 1000,
+        tags: [],
+        participants: [],
+      },
+      {
+        id: 4,
+        host: 0,
+        creationTime: '',
+        lastUpdatedTime: '',
+        schedule: [],
+        title: 'When Space X going to launch a rocket to the Mars?',
+        description: 'Let\'s analyse the probability!',
+        price: 999999.99,
+        maxParticipants: 1000,
+        tags: [],
+        participants: [],
+      },
+      {
+        id: 5,
+        host: 0,
+        creationTime: '',
+        lastUpdatedTime: '',
+        schedule: [],
+        title: 'title1',
+        description: 'description1',
+        price: 999999.99,
+        maxParticipants: 1000,
+        tags: [],
+        participants: [],
+      },
+      {
+        id: 6,
+        host: 0,
+        creationTime: '',
+        lastUpdatedTime: '',
+        schedule: [],
+        title: 'title2',
+        description: 'description2',
+        price: 999999.99,
+        maxParticipants: 1000,
+        tags: [],
+        participants: [],
+      },
+      {
+        id: 7,
+        host: 0,
+        creationTime: '',
+        lastUpdatedTime: '',
+        schedule: [],
+        title: 'title3',
+        description: 'description3',
+        price: 999999.99,
+        maxParticipants: 1000,
+        tags: [],
+        participants: [],
+      },
+      {
+        id: 8,
+        host: 0,
+        creationTime: '',
+        lastUpdatedTime: '',
+        schedule: [],
+        title: 'title4',
+        description: 'description4',
+        price: 999999.99,
+        maxParticipants: 1000,
+        tags: [],
+        participants: [],
+      },
+    ];
+
+    let ret = [];
+    let max_size = Math.min(events_list.length, pageSize);
+    max_size = Math.min(max_size, 9);
+    ret = events_list.slice(0, max_size);
+
+    return {
+      cursor: 8,
+      events: ret,
+    };
   },
-  event: async (_, {id}) => {
+  event: async (_, {id}) => { },
+  me: async (_, __, context) => { },
+  user: async (_, {id}) => { },
+  userEvents: async (_, {info}, {dataSources, userId}) => {
```

Comment:
"info" is not a well defined variable name. Also, how do we compute upcoming vs past? Shouldn't that be computed based on current time?

Reply:
> "info" is not a well defined variable name. Also, how do we compute upcoming vs past? Shouldn't that be computed based on current time?

Thank you. You are correct, we computed upcoming vs past based on current time in API(getUserUpcomingEvents or getUserPastEvents). And i modified variable name.